src/test/tools/pmemobjcli/pmemobj_allocate.posc

```diff
@@ -0,0 +1,61 @@
+#!./pmemobjcli -s
+# usage: ./pmemobj_allocate.posc <file>
```

Comment:
In the original example there was a comment with description, maybe it's worth adding here?

And since we're introducing new examples and there might be more of them in the future, maybe we should add a sub-folder with examples?